libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxDistanceFormatter.kt

```diff
@@ -136,25 +120,28 @@ class MapboxDistanceFormatter(
      * @param unit string from TurfConstants. This will be converted to the abbreviated form.
      * @return String with bolded distance and shrunken units
      */
-    private fun getDistanceString(distance: String, unit: String): SpannableString {
-        val spannableString = SpannableString("$distance ${unitStrings[unit]}")
+    fun getSpannableDistanceString(distanceAndSuffix: Pair<String, String>): SpannableString {
```

Comment:
Does this need to be `public`? I know we're calling it from tests, but can't we run those tests calling the public API `formatDistance` and leaving this as `private`?

Reply:
Since it's a pure function I saw an advantage in leaving it public because I can test it directly. I don't see any advantage in hiding in the class. If it mutated state I'd definitely make it private but since it doesn't it's an advantage to test it directly IMO.

Reply:
Maybe `internal` then?